web-ui/src/main/resources/catalog/templates/admin/dashboard/status.html

```diff
@@ -140,6 +140,29 @@ <h4 class="list-group-item-heading" translate> {{info.name}} </h4>
         </ul>
       </div>
     </div>
+  </div>  
+  <div class="panel-group" id="logActivityTop" role="tablist">
+    <div class="panel panel-default">
+      <div class="panel-heading" role="tab" id="logActivityHeading">
+        <h4 class="panel-title">
+          <a data-toggle="collapse" data-ng-click="openLogActivity()" data-parent="#top" aria-expanded="true" aria-controls="logActivity" translate>
+              logActivity
+          </a>&nbsp;
+          <i class="fa fa-spinner fa-spin" data-ng-show="logInfoLoading" ></i>
+        </h4>
+        <div class="btn-toolbar">
+	        <a href="log/file" class="btn btn-default pull-right" >
+	      		<i class="fa fa-file-zip-o"></i>&nbsp;<span translate>exportMEF</span>
```

Comment:
You should not use the exportMEF here as MEF is Metadata Exchange Format and can't apply to a log file in ZIP format. Create a new one, eg. exportLogAsZIP